src/components/chat/ChatIframe.tsx

```diff
@@ -0,0 +1,93 @@
+import grill, { GrillConfig, GrillEventListener } from '@subsocial/grill-widget'
+import { Resource } from '@subsocial/resource-discussions'
+import { summarize } from '@subsocial/utils'
+import clsx from 'clsx'
+import { ComponentProps, useEffect } from 'react'
+import useWrapInRef from 'src/hooks/useWrapInRef'
+import { useSendEvent } from 'src/providers/AnalyticContext'
+import { useSetChatTotalMessageCount } from 'src/rtk/app/hooks'
+import { useAppSelector } from 'src/rtk/app/store'
+import { ChatEntity } from 'src/rtk/features/chat/chatSlice'
+
+export type ChatIframeProps = ComponentProps<'div'> & {
+  onUnreadCountChange?: (count: number) => void
+}
+
+const COMMENTS_HUB_ID = '1032'
```

Comment:
Extract it to config

Reply:
Updated, moved it into
`common` config with name `commentsHubId`